docs/source/tuning_strategies.md

```diff
@@ -114,38 +145,52 @@ conf = PostTrainingQuantConfig(
 
 ### Design
 
-The `Basic` strategy is designed for quantizing most models. There are three stages executed by `Basic` strategy sequentially, and the tuning process ends once the condition meets the exit policy. 
+The `Basic` strategy is designed for quantizing most models. There are six stages executed by `Basic` strategy sequentially, and the tuning process ends once the condition meets the exit policy. 
 
-- **Stage I**. OP Type Wise Tuning
+- **Stage I**. Quantize with default quantization configuration
+    
+    At this stage, it tries to quantize OPs with the default quantization configuration which is consistent with the framework behavior.
+
+- **Stage II**. Apply all recipes
+
+    At this stage, it tries to apply all recipes.
+
+- **Stage III**. OP-Type-Wise Tuning
 
-    In this stage, it tries to quantize the OPs as many as possible and traverse all OP type wise tuning configs. Note that,  the OP is initialized with different quantization modes according to the quantization approach.
+    At this stage, it tries to quantize OPs as many as possible and traverse all OP type wise tuning configs. Note that, the OP is initialized with different quantization modes according to the quantization approach.
     
     a. `post_training_static_quant`: Quantize all OPs support PTQ static.
 
     b. `post_training_dynamic_quant`: Quantize all OPs support PTQ dynamic.
     
     c. `post_training_auto_quant`: Quantize all OPs support PTQ static or PTQ dynamic. For OPs supporting both PTQ static and PTQ dynamic, PTQ static will be tried first, and PTQ dynamic will be tried when none of the OP type wise tuning configs meet the accuracy loss criteria.
 
-- **Stage II**. Fallback OP One by One
+- **Stage IV**. Try recipe One by One
 
-    In this stage, it performs high-precision OP (FP32, BF16 ...) fallbacks one by one based on the tuning config with the best result in the previous stage, and records the impact of each OP. 
+    At this stage, it tries recipe one by one based on the tuning config with the best result in the previous stage.
 
-- **Stage III**. Fallback Multiple OPs Accumulated
+- **Stage V**. Fallback OP One by One
 
-    In the final stage, it first sorted the OPs list according to the impact score in stage II, and tries to incrementally fallback multiple OPs to high precision according to the sorted OP list.
+    At this stage, it performs high-precision OP (FP32, BF16 ...) fallbacks one by one based on the tuning config with the best result in the previous stage, and records the impact of each OP. 
+
+- **Stage VI**. Fallback Multiple OPs Accumulated
+
+    At the final stage, it first sorted the OPs list according to the impact score in stage II, and tries to incrementally fallback multiple OPs to high precision according to the sorted OP list.
```

Comment:
the impact score in Stage II? 
Based on the above description, it looks like Stage V.